### ISSUE TITLE
v12 release still requires angular v8 as a peer dependency

### ISSUE THREAD

**Issue by Druue** [orizens/ngx-youtube-player#84]
I'm wondering if this is intended, despite the main package.json listing a dependency of angular v12
```
Could not resolve dependency:
peer @angular/common@"^8.2.0" from ngx-youtube-player@12.0.1
```

**Comment by anees-tricon** [comment]
@orizens can we please fix this.

https://github.com/orizens/ngx-youtube-player/pull/95 is an attempt to fix this. Please merge if fit

**Comment by orizens** [comment]
fixed by #95
